components/dashboard/package.json

```diff
@@ -26,6 +26,7 @@
     "idb-keyval": "^6.2.0",
     "js-cookie": "^3.0.1",
     "lodash.debounce": "^4.0.8",
+    "lucide-react": "^0.287.0",
```

Comment:
Oh, cool! I know a lot of the icons in use currently are Heroicons, but why we've never installed them as a dep I am unsure about. 

The only fear I have here then is that we use 2 of the same icons across different design systems. It's a bit hard to keep track of, but maybe replacing the regular SVGs over time sounds like a good idea.

Reply:
Yah, this one was the easiest to plug in here, lots are quite generic but we can change this to something else if we want. It might be nice to have a layer of abstraction of icons, but feels a bit too early to do that.